source/index.html.md

```diff
@@ -1756,6 +1757,32 @@ print(chain.active_chain.api.GetTransaction("655107c300e86ee6e819af1cbfce097db15
 }
 ```
 
+*Note: If the transaction had an data field or code field, it will be displayed*
```

Comment:
can you provide a sample output that has data or code field?

Reply:
it is there, there are two examples